src/main.js

```diff
@@ -27,6 +29,10 @@ router.beforeEach((to, from, next) => {
     next()
 });
 
+String.prototype.capitalize = function() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [String prototype is read only, properties should not be added.](https://app.codacy.com/gh/FAIRsharing/fairsharing.github.io/pullRequest?prid=4811466)